internal/server/server.go

```diff
@@ -168,14 +169,17 @@ func RunServer(config *Config) error {
 			},
 		}),
 	)
-	address := fmt.Sprintf(":%d", config.Port)
+
+	// grpcPort has to be same as config.Port till the time guardian service can support both grpc and http in two different ports
+	grpcPort := config.Port
+	address := fmt.Sprintf(":%d", grpcPort)
```

Comment:
in this case, can we remove `GRPCConfig.Port` for now since it's not being used anywhere?

Reply:
done